datasets/case01/queries/0001.1_fetchObjectByIdIN.sql

```diff
@@ -0,0 +1,7 @@
+-- Find objects with a particular object ids
+-- http://dev.lsstcorp.org/trac/wiki/dbQuery009
+
+-- pragma sortresult
+select ra_Ps, decl_PS
+FROM Object
+WHERE objectId IN(390034570102582, 396210733076852, 393126946553816, 390030275138483)
```

Comment:
Could you add a space after `IN`, it should make it slightly more readable. Same thing below.